tests/leds/main.c

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (C) 2016 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser General
+ * Public License v2.1. See the file LICENSE in the top level directory for more
+ * details.
+ */
+
+/**
+ * @ingroup     tests
+ * @{
+ *
+ * @file
+ * @brief       Test for the on-board LED macros
+ *
+ * @author      Hauke Petersen <hauke.petersen@fu-berlin.de>
+ *
+ * @}
+ */
+
+#include <stdio.h>
+
+#include "board.h"
+#include "xtimer.h"
+
+#define WAIT_INTERVAL       (500 * MS_IN_USEC)
+
+int main(void)
+{
+    puts("On-board LED test\n");
+    puts("You should now see the 'red' and the 'green' LED lighting up in a\n"
```

Comment:
I didn't accuse you to make a false statement here. :D
I just have been wondering if this may be more confusing than clarifying for inexperienced users.